source/extensions/omni.isaac.lab/test/sensors/test_frame_transformer.py

```diff
@@ -299,8 +315,174 @@ def test_frame_transformer_feet_wrt_thigh(self):
                     source_pose_w_gt[:, :3], source_pose_w_gt[:, 3:], feet_pos_w_tf[:, index], feet_quat_w_tf[:, index]
                 )
                 # check if they are same
-                torch.testing.assert_close(feet_pos_source_tf[:, index], foot_pos_b, rtol=1e-3, atol=1e-3)
-                torch.testing.assert_close(feet_quat_source_tf[:, index], foot_quat_b, rtol=1e-3, atol=1e-3)
+                torch.testing.assert_close(feet_pos_source_tf[:, index], foot_pos_b)
+                torch.testing.assert_close(feet_quat_source_tf[:, index], foot_quat_b)
+
+    def test_frame_transformer_body_wrt_cube(self):
```

Comment:
How is this test different from previous tests where we compute base to some other rigid body? Was this case also not working before?

Reply:
This frame is external to the articulation, which previously didn't work

Reply:
NIT: Can you please fix the test names so there are more intuitive and clear to understand? Right now `test_frame_transformer_body_wrt_cube` doesn't seem to make a lot of sense.